sway/main.c

```diff
@@ -25,10 +28,6 @@ int main(int argc, char **argv) {
 	}
 	setenv("DISPLAY", ":1", 1);
 
-	if (!load_config()) {
-		sway_abort("Unable to load config");
-	}
-
```

Comment:
Ah I see. I assume it would be ok to just run `sway_abort` at the end of the main function if `load_config` failed?